### DISCUSSION TITLE
Tenant Factory correct use

### DISCUSSION THREAD

**Discussion by rossiellog** [GoogleCloudPlatform/cloud-foundation-fabric#2672] [Q&A] [answered]
Hello everyone!
I am currently using FAST to build a foundation for a project. For this project I am considering using the tenant factory stage, as the project needs separate tenants for different countries. Since these tenants do not need a high degree of separation (e.g. they'll still use the common networking at org level), simple tenants seem to be enough.
I have run the resman stage to create a Countries folder under the org, and my intention is to place tenants under this folder. When I run the tenant stage, terraform proposes to destroy some gcs bucket objects, namely providers for the next stages (i.e. networking, security, etc.). After quickly investigating this, it appears that, as of the current version of fast, the resman and tenant stages are expected to share the same terraform state. Why is that? How can I avoid one stage to override some of the contents of the other?
Thanks!

**Comment by rossiellog** [comment, marked as answer]
I am sorry, I asked the question too soon... I realized that the providers template I was using was not the latest one, so the backend_extra parameter was not taken in consideration. Now that the template has been updated, the state prefix makes it so the states will not overlap.

**Reply by ludoo** [reply]
thanks for circling back and sorry if we lagged a bit in replying :)
